### DISCUSSION TITLE
Support for nRF52 DK (nRF52832)

### DISCUSSION THREAD

**Discussion by usmanmehmood55** [jrowberg/i2cdevlib#597] [Q&A]
Hi. 
Can the library work with nRF52 DK? It uses nRF52832

**Comment by jrowberg** [comment]
If you're using the native nRF52 SDK, then no, at least not without a great deal of porting. If you're using Adafruit's nRF52832-based board with the Arduino support they've added, and building in the Arduino IDE, then I expect it will work; however, I have never tried using it in that context, and there may be implementation issues. I know that it will at least compile without errors when the Adafruit Feather nRF52832 board is installed and selected in the IDE.
